cvat-ui/src/components/reset-password-confirm-page/reset-password-confirm-form.tsx

```diff
@@ -0,0 +1,156 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+import Form, { FormComponentProps } from 'antd/lib/form/Form';
+import Button from 'antd/lib/button';
+import Icon from 'antd/lib/icon';
+import Input from 'antd/lib/input';
+
+import patterns from 'utils/validation-patterns';
+
+export interface ResetPasswordConfirmData {
+    newPassword1: string;
+    newPassword2: string;
+    uid: string;
+    token: string;
+}
+
+type ResetPasswordConfirmFormProps = {
+    fetching: boolean;
+    onSubmit(resetPasswordConfirmData: ResetPasswordConfirmData): void;
+} & FormComponentProps & RouteComponentProps;
+
+class ResetPasswordConfirmFormComponent extends React.PureComponent<ResetPasswordConfirmFormProps> {
+    private validateConfirmation = (_: any, value: string, callback: Function): void => {
+        const { form } = this.props;
+        if (value && value !== form.getFieldValue('newPassword1')) {
+            callback('Two passwords that you enter is inconsistent!');
```

Comment:
`do not match` would be better, i guess.

Reply:
+1 `Passwords do not match`